lib/miq_rubyrep.rb

```diff
@@ -48,6 +49,7 @@ def self.prepare_configuration(config)
     config.options[:right_record_handling]         = :ignore
     config.options[:left_record_handling]          = :insert
     config.options[:sync_conflict_handling]        = :left_wins
+    config.options[:heartbeat_file]                = rp_conf.fetch_path(:options, :heartbeat_file) || defaults.fetch_path(:options, :heartbeat_file)
```

Comment:
Wondering if we should just hardcode it here and use `Rails.root.join("tmp/rubyrep_heartbeat")` or something.  Then it's not needed in configuration either.  You *could* make it a constant here in case we need it in other Ruby code.